app/components/appcomponents/MemorialPageView.js

```
@@ -1,132 +1,149 @@
-                className="h-[190.11px] w-[136px]  rounded-lg"
-              />
-            </div>
-            <div className="text-center mt-3 text-[14px] font-normal text-[#1E2125] leading-[24px]">
-              Danilo Primo
+            >
+              {obituaries?.map((item, index) => {
+                if (index < 3) {
+                  const imageUrl = item.image ? `${API_BASE_URL}/${item.image}` : "/user5.jpeg";
+                  return (
+                    <div className="mobile:pr-5 mx-5">
+                      <div className="rounded-xl shadow-custom-light-dark-box-image p-1 bg-gradient-to-br from-[#E3E8EC] to-[#FFFFFF] ">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)